package.json

```diff
@@ -5,7 +5,7 @@
   "main": "bot.js",
   "scripts": {
     "start1": "node bot.js",
-    "start": "xvfb-run node bot.js",
```

Comment:
Hey Tom, please bring back `xvfb-run`. The prod env is headless, which prevents the scraper from working as it does on a dev laptop, unless xvfb-run is used:
https://github.com/tiagosiebler/Jarvis/blob/master/skills/search.js#L2

Reply:
Whoops....will do!